src/components/DialogContentContainer/__stories__/DialogContentContainer.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import DialogContentContainer from "../DialogContentContainer";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import DialogContentContainerExample from "./DialogContentContainerExample";
+import "./DialogContentContainer.stories.scss";
+
+<Meta title="Popover/DialogContentContainer" component={DialogContentContainer} />
+
+# DialogContentContainer
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Feedback](#feedback)
+
+## Overview
+
+Please write here the component description
+
+<Canvas>
+  <Story name="Overview">
```

Comment:
when we define the story like this I think the "show code" section is not valuable (we see the DialogContentContainerExample in the code)
Why not define the template function (by using the helper or not if its more complex case) in the jsx file and import it to here?